torch/_torch_docs.py

```diff
@@ -8287,10 +8292,16 @@ def merge_dicts(*dicts):
            r"""
 remainder(input, other, *, out=None) -> Tensor
 
-Like :func:`torch.fmod` this applies C++'s `std::fmod <https://en.cppreference.com/w/cpp/numeric/math/fmod>`_
-for floating point tensors and the modulus operation for integer tensors.
-Unlike :func:`torch.fmod`, however, if the sign of the modulus is different
-than the sign of the divisor :attr:`other` then the divisor is added to the modulus.
+Computes
+`Python's modulus operation <https://docs.python.org/3/reference/expressions.html#binary-arithmetic-operations>`_
```

Comment:
Why the newline?

Reply:
The linter was complaining, and I did not want to figure out what's the correct way to split that hyperlink in sphinx :P

Reply:
ok!